metrics/ci_integrations/lib/source/jenkins/adapter/jenkins_source_client_adapter.dart

```diff
@@ -243,6 +254,21 @@ class JenkinsSourceClientAdapter with LoggerMixin implements SourceClient {
     }
   }
 
+  /// Returns a [Duration] of a [JenkinsBuild] depending on the given
+  /// [buildStatus] and [duration].
+  ///
+  /// Returns `null` if the given [buildStatus] is [BuildStatus.inProgress].
+  /// Otherwise, returns the given [duration] if it is not `null` or
+  /// [Duration.zero] if the given [duration] is `null`.
+  Duration _mapJenkinsBuildDuration(
+    BuildStatus buildStatus,
+    Duration duration,
+  ) {
```

Comment:
```suggestion
  /// Processes the given [duration] of a build depending on the [status] of this build.
  ///
  /// Returns `null` if the given [status] is [BuildStatus.inProgress].
  /// Otherwise, returns either the given [duration] or [Duration.zero] (if the given [duration] is `null`).
  Duration _processBuildDuration(
    BuildStatus status,
    Duration duration,
  ) {
```
Not sure the `map` word in the method name is correct as this is not actually mapping and more similar to processing.